docs/ENV.MD

```diff
@@ -1,29 +1,29 @@
 # Environment variables
 This guide explains all the available configurations in your `.env` file so that you can configure your environments accordingly.
 
-## VUE_APP_PLIO_AF_API_KEY
+## VITE_APP_PLIO_AF_API_KEY
```

Comment:
This change is not needed. It's still a Vue app, just using Vite

Reply:
It's a Vue app but the bundler is Vite. Before it was Vue CLI, so to read the env variables they needed to start with VUE_APP_. Now for Vite to read them, they need to start with VITE_APP